packaging/aznfs/DEBIAN/postrm

```diff
@@ -8,5 +8,8 @@
 if [ "$1" == 'remove' -o "$1" == 'purge' ]; then
 	chattr -f -i /opt/microsoft/aznfs/data/mountmap
 	chattr -f -i /opt/microsoft/aznfs/data/randbytes
+	chattr -f -i /opt/microsoft/aznfs/data/mountmapv4
 	rm -rf /opt/microsoft/aznfs
-fi
\ No newline at end of file
+	chattr -f -i /etc/stunnel/microsoft/aznfs/nfsv4_fileShare/stunnel*
```

Comment:
When was this made immutable? If this is not immutable then you don't need this. 

For both deb and rpm.

Reply:
It's the stunnel_conf_file which is made immutable in nfsv4mountscript.sh.